pkgs/servers/home-assistant/custom-components/luxer_one/package.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildHomeAssistantComponent,
+  ruff,
+}:
+
+buildHomeAssistantComponent rec {
+  owner = "tjhorner";
+  domain = "luxer";
+  version = "0-unstable-2023-05-06";
+
+  src = fetchFromGitHub {
+    owner = "tjhorner";
+    repo = "home-assistant-luxer-one";
+    rev = "f6a810034ab76e6a8635de755c4a1750e86b1674";
+    hash = "sha256-WmsL0NLe2ICqNGbEQ4vg1EzcZgIGi++G9aDyKjnmJMs=";
+  };
+
+  dependencies = [
+    ruff
+  ];
```

Comment:
Ruff is a linter/formatter. If you've got it from [requirements.txt](https://github.com/tjhorner/home-assistant-luxer-one/blob/main/requirements.txt), those are dev dependencies.

You are looking for `requirements` in the [manifest.json](https://github.com/tjhorner/home-assistant-luxer-one/blob/main/custom_components/luxer/manifest.json#L13), which for this component is empty because it reuses the aiohttp library that home-assistant already depends on.

```suggestion
```

Reply:
Ah thanks, made the change :)